template/config/index.js

```diff
@@ -7,7 +7,19 @@ module.exports = {
     index: path.resolve(__dirname, '../dist/index.html'),
     assetsRoot: path.resolve(__dirname, '../dist'),
     assetsSubDirectory: 'static',
-    assetsPublicPath: '/',
+    assetsPublicPath: '/',{{#cdn}}
+    packages: [
+      {
+        name: 'vue',
+        var: 'Vue',
+        path: 'dist/vue.runtime.min.js'
+      }{{#router}},
```

Comment:
I'm confused, thought the `{{#router}}` switch is meant to do that? Are you suggesting to actually read the packages.json file directly and not by flag?